src/sentry/integrations/slack/notifications.py

```diff
@@ -75,69 +70,39 @@ def _notify_recipient(
 
         text = notification.get_notification_title(ExternalProviders.SLACK, shared_context)
 
-        # NOTE(isabella): we check that attachments consists of blocks in case the flag is turned on
-        # while a notification with the legacy format is being sent
-        if features.has("organizations:slack-block-kit", notification.organization) and isinstance(
-            local_attachments, dict
-        ):
```

Comment:
do we still need the type check?

Reply:
no, since we always send blocks now we don't need to check if we're in the middle of sending an attachments payload